chrome/skin/default/zotero/itemPane.css

```diff
@@ -67,3 +67,29 @@
 {
 	min-height: 20px;
 }
+
+#zotero-feed-item-toggleRead-button {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	max-width: 150px;
+}
+
+#zotero-feed-item-addTo-button {
+	max-width: 250px;
+}
+
+#zotero-feed-item-addTo-button button {
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+
+/* Set to hidden in user-agent css for some reason. */
+#zotero-feed-item-addTo-button .button-icon {
+	display: block;
+	margin-right: 5px
```

Comment:
These can be removed as per below, but do need to set `height: 16px;` here to prevent double-size icons on HiDPI OS X.